lib/util.mli

```diff
@@ -58,3 +58,17 @@ module String : sig
   (** [english_conjonction ["a"; "b"; "c"]] returns ["a, b and c"].
       @raise Invalid_argument when called on the empty list. *)
 end
+
+module Sexp : sig
```

Comment:
Would depending on `sexplib` be bad ?
Getting sexp right is more than a dozen of lines.